LICENSE.bin

```diff
@@ -379,6 +379,14 @@
    npgall concurrent-trees
    reyerizo jcstress-gradle-plugin
    melix jmh-gradle-plugin
+   Lance
+   JCasbin
+   OGNL
+   Apache Flink
+   Apache Flink JDBC
+   Google FlatBuffers
+   Google FindBugs
```

Comment:
Why do we include find bugs jar in the binary release? It should be used for developemnt?